cli/src/main.rs

```diff
@@ -606,10 +603,7 @@ fn send_direct_request(
 
 	debug!("setup sender and receiver");
 	let (sender, receiver) = channel();
-	match direct_api.watch(jsonrpc_call, sender) {
-		Ok(_) => {},
-		Err(_) => panic!("Error when sending direct invocation call"),
-	}
+	direct_api.watch(jsonrpc_call, sender);
```

Comment:
May I ask for the background behind that change?

Reply:
Because I had to adapt the rpc-client (added two new rpc calls) I saw the following:
https://github.com/integritee-network/worker/blob/master/core/rpc-client/src/direct_client.rs#L131-L153

Which I probably implemented myself, but is, sorry for that, bullshit. An Error is never returned. Now it's atleast visible.

Reply:
Haha, I see. Thanks for the explanation.